src/Uno.UI/UI/Xaml/Controls/Image/Image.Android.cs

```diff
@@ -309,6 +311,12 @@ private async Task SetSourceResource(ImageSource newImageSource)
 			Func<CancellationToken, Task> setResource = async (ct) =>
 			{
 				_native.SetImageResource(newImageSource.ResourceId.Value);
+
+				if (MonochromeColor != null)
```

Comment:
There might be a better way/place to do this that I wasn't able to find, but (for now?) from what I gather, _native's base.SetImageResource handles all of the actual displaying given the source, including bitmap generation, so I had to wait for it to be called to recolor the bitmap and send it back. Insight would be appreciated

Reply:
It's not terrible to have it here, but couldn't you do it in `NativeImage.SetImageResource()`?